src/components/OrderList/OrderList.tsx

```diff
@@ -22,6 +22,7 @@ const OrderList: React.SFC<OrderListProps> = ({
       <tbody>
         {orders.map(order => {
           const instrument = getInstrumentById(order.symbol);
+          order.accuracy = instrument.quoteAsset.accuracy;
```

Comment:
this is a very bad practice to mutate state during render. i even don't know in all details how mobx deals with that, but the thing is that this is completely unneeded, see my comments about `value` and formatting